s2i/bin/assemble

```diff
@@ -42,6 +42,10 @@ if [ -f composer.json ]; then
       chmod g+w web/sites/default
   fi
 
+  echo "---> Storing git ref..."
+  mkdir -p web/.shepherd
+  git rev-parse HEAD > web/.shepherd/git-ref
+
```

Comment:
Might be nice to add `git describe --tags HEAD`
I.e. list all tags related to the current commit.